report/report.tex

```diff
@@ -108,19 +110,39 @@ \section{Problem Formulation}
 %------------------------------------------------
 \section{Methods}
 
-Maecenas sed ultricies felis. Sed imperdiet dictum arcu a egestas.
-\begin{itemize}
-	\item Donec dolor arcu, rutrum id molestie in, viverra sed diam
-	\item Curabitur feugiat
-	\item turpis sed auctor facilisis
-	\item arcu eros accumsan lorem, at posuere mi diam sit amet tortor
-	\item Fusce fermentum, mi sit amet euismod rutrum
-	\item sem lorem molestie diam, iaculis aliquet sapien tortor non nisi
-	\item Pellentesque bibendum pretium aliquet
-\end{itemize}
-\blindtext % Dummy text
+\subsection{Data exploration}
+
+With the cleaned up dataset and the research question in mind*, columns with relevance are explored. To explore, we plot different columns of interest to identify any patterns in the data. We plot the distribution between reviews with votes and no votes as well as review rating.
+\begin{figure}[h]
+	\centering
+	\includegraphics[width=0.5\textwidth]{img/no_votes_vs_votes.pdf}
+	\caption{Vote distribution}
+	\label{fig:votes_distribution}
+\end{figure}
+As seen in \figurename{\ref{fig:votes_distribution}}, the distribution is very uneven in favor of reviews with no votes. This is relevant when evaluating our predictive power. To get another perspective on review votes we plot the data into a heatmap in relation to ratings.
+\begin{figure}[h]
+	\centering
+	\includegraphics[width=0.5\textwidth]{img/rating_vote_heatmap_excluding_no_votes.pdf}
+	\caption{Votes in relation to rating}
+	\label{fig:rating_vote_heatmap}
+\end{figure}
+This shows that there is a drastic change after 11 upvotes.
```

Comment:
Måske forklare lidt om, hvorfor det mon er sådan? Hvis det er muligt :D

Reply:
Har givet et eksempel paa noget men ved ikke om det giver mening at have med, siden det er ren spekulation.

Reply:
Hmm. Vi skal nok spørge os selv, om plottet så er relevant for vores rapport.

Reply:
Visualiserer den noget som ville være svært at forstå uden?

Reply:
Det er måske overkill at have dem alle, da heatmappet er en kombi af mange af dem

Reply:
Bare husk at forklar at heatmappet er uden dem med ingen votes

Reply:
Yeah nok bedst at begraense antallet af grafer. Er ogsaa mest fan af votes/no votes og heatmappet.

Reply:
Vi kan dog ikke sige noget om hvorfor der er et drastisk fald efter 11 upvotes

Reply:
Har skrevet teksten om, så tag et kig når I har tid :D